common/src/main/java/io/crate/types/TimestampType.java

```diff
@@ -151,6 +154,16 @@ static long parseTimestamp(String timestamp) {
         }
     }
 
+    public static long addPeriod(Long timestamp, Period period) {
```

Comment:
I'd inline these methods given that they're very shallow, and the indirection probably causes more complexity than it encapsulates. Or at least move to `sql` where they're used.